tests/test_frontend_op_resolver.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
```

Comment:
Can you also add tests that actually work? :)

Reply:
Haha indeed!